tests/integration/kubernetes/gha-run.sh

```diff
@@ -241,6 +241,16 @@ install_kata_tools_placeholder() {
 		"after CI picks up the gha yaml changes required to test that installation."
 }
 
+function deploy_snapshotter() {
+	echo "::group::Deploying ${SNAPSHOTTER}"
+	echo "::endgroup::"
+}
+
+function cleanup_snapshotter() {
+	echo "::group::Cleanuping ${SNAPSHOTTER}"
```

Comment:
Same here.

Reply:
Fixed.

Reply:
@fidencio , @ChengyuZhu6 I'm flagging this thread as solved to facilitate on our eyes during the review. Please feel free to reopen if need.